### ISSUE TITLE
'WebClient' object has no attribute 'assistant_threads_setStatus' 

### ISSUE THREAD

**Issue by MadhabaPatra** [slackapi/bolt-python#1174]
Hi Team,

Here are the Slack packages we've been using so far:
- slack-bolt==1.18.1
- slack_sdk==3.27.2

This week, we learned about Agents & Assistants, which is quite new to us. However, the [documentation](https://api.slack.com/docs/apps/ai#respond-assistant-thread-started-event) has been very helpful in getting us started.

We’re currently exploring the [assistant.threads.setStatus](https://api.slack.com/methods/assistant.threads.setStatus) feature. We're trying to implement the following code:
```
client = WebClient(token=bot_token)
client.assistant_threads_setStatus ---> Throwing a 'no attribute' error.
```

Are we missing something here?

**Comment by seratch** [comment]
Hi @MadhabaPatra, thanks for writing in! The method is available since v3.33.0. Once you upgrade slack-sdk to the version or newer, you can use it. Also, checking this template https://github.com/slack-samples/bolt-python-assistant-template should be helpful for you!

**Comment by MadhabaPatra** [comment]
Thanks @seratch , I was focused on tracking the Bolt library package and missed the slack-sdk version update. I'll upgrade to the recommended version and check out the template you've shared. Appreciate the help!
